org.eclipse.buildship.oomph/.classpath

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src/main/java-generated"/>
+	<classpathentry kind="src" path="src/main/resources"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins">
+		<accessrules>
+			<accessrule kind="accessible" pattern="org/eclipse/oomph/**"/>
```

Comment:
Remove, maybe?

Reply:
Hmm, the initial project creation added that one, not sure why or what for. I've removed those access rules from the .classpath files.

Reply:
Turns out these access rules are necessary so that the generated code can access oomph classes and methods without warnings.

Reply:
Thanks for checking. I just wanted to make sure it's necessary to have those rules.